### PR TITLE
www.istikbalgazetesi.com

### PR DESCRIPTION
**Pull request by gork7777**
issue: #104789

### CODE REVIEW

File: TurkishFilter/sections/specific.txt [hunk 1]
```diff
@@ -45,6 +45,9 @@ megahaber27.com,yemektarifleri.org,lifebursa.com,mshowto.org,macizlee1.com,maciz
 ! ###adStarter
 fullhdfilmcibaba.com,720pizlemek.com,hdfilmizledayi.com,fullhdfilmizlebaba.com###adStarter
 !
+istikbalgazetesi.com##.side-banner
+||istikbalgazetesi.com/anadolu.jpg
```
Comment (Sergey-Lyapin) [inline TurkishFilter/sections/specific.txt:49]: Let's use `istikbalgazetesi.com##div[align="center"] > a[target="_blank"] > img` instead these 2

**Review by Sergey-Lyapin — CHANGES_REQUESTED** [review]

File: TurkishFilter/sections/specific.txt [hunk 2]
```diff
@@ -45,6 +45,8 @@ megahaber27.com,yemektarifleri.org,lifebursa.com,mshowto.org,macizlee1.com,maciz
 ! ###adStarter
 fullhdfilmcibaba.com,720pizlemek.com,hdfilmizledayi.com,fullhdfilmizlebaba.com###adStarter
 !
+istikbalgazetesi.com##div[align="center"] > a[target="_blank"] > img
+||istikbalgazetesi.com/gurlife.jpg
```
Comment (Sergey-Lyapin) [inline TurkishFilter/sections/specific.txt:49]: please remove this rule and add:
`istikbalgazetesi.com##.side-banner`

Comment (gork7777) [reply TurkishFilter/sections/specific.txt:49]: Should I remove `||istikbalgazetesi.com/gurlife.jpg` ?

Comment (Sergey-Lyapin) [reply TurkishFilter/sections/specific.txt:49]: @gork7777 yes

**Pull request merged**